pkg/kubelet/pod_workers.go

```diff
@@ -819,6 +857,44 @@ func (p *podWorkers) acknowledgeTerminating(pod *v1.Pod) PodStatusFunc {
 	return nil
 }
 
+// queueReservedWork queues a reserved work having the same full name as pod
+// and completes the previous pod. It returns true if there is a reserved work
+// else returns false
+func (p *podWorkers) queueReservedWork(pod *v1.Pod) bool {
```

Comment:
```suggestion
func (p *podWorkers) queueWorkForConflictedStaticPod(pod *v1.Pod) bool {
```

This currently only applies to a small set of pods, lets avoid having a generic name that's easy to piggyback onto in the future without clear intent